package.json

```diff
@@ -14,7 +14,7 @@
   "module": "./web/noir_wasm.js",
   "sideEffects": false,
   "peerDependencies": {
-    "@noir-lang/noir-source-resolver": "1.1.2"
+    "@noir-lang/noir-source-resolver": "1.1.1"
```

Comment:
Looks like this pulled from upstream, so we need to remove the change completely